src/cf-layout.less

```diff
@@ -1404,6 +1404,289 @@
+            .hero .hero_wrapper:after {           
+              background-size: 150px;
+              width: 195px;
+            }
+          }
+        </style>
+        <section class="content_hero hero hero__small-content">
+            <div class="hero_wrapper wrapper wrapper__match-content">
+                <div class="hero_text">
+                    <h1 class="hero_heading">Hero title</h1>
+                    <p class="hero_subhead">
+                        Hero text goes here
+                    </p>
+                </div>
+            </div>
+        </section>    
+      notes:
+        - "If the hero text is short or doesn't include a call to action, adding
+           the small-content class will increase the top padding value so the
+           text is more vertically aligned with the image."
+      codenotes:
+        - .hero__small-content
+  tags:
+    - cf-layout
+*/
+
+@hero-desktop-height: 300px;
+@hero-image-height: 195px;
+@desktop-image-container-width: 475px; //width of 5 of 12 columns at 1140px
+@tablet-image-container-width: 333px; //width of 5 of 12 columns at 799px
+
+.hero {
+  background-color: @gray-10;
+  
+  &_wrapper {
+    position: relative;
+    padding-top: unit(@grid_gutter-width / @base-font-size-px, em);
+    // bottom padding = image height at mobile + padding for top/bottom of image
+    padding-bottom: unit((@hero-image-height + @grid_gutter-width * 2) / @base-font-size-px, em); 
+    
+    &:after {
+      display: block;
+      width: 100%;
+      height: 100%;
+      position: absolute;
+      top:0;
+      right: 0;
+      content: "";
+      background-position: center bottom unit(@grid_gutter-width / @base-font-size-px, em);
+      background-repeat: no-repeat;
+    }
+
+    .respond-to-min(@tablet-min, {
+      padding-right: unit(@tablet-image-container-width / @base-font-size-px, em);
+      padding-bottom: unit(@grid_gutter-width / @base-font-size-px, em);
+      
+      &:after {
+        width: @tablet-image-container-width;
+        background-position: center;
+      }
+    });
+
+    .respond-to-min(800px, {
+      box-sizing: border-box;
+      min-height: @hero-desktop-height;
+      padding-top: unit(40px / @base-font-size-px, em); // approximates 45px visually
+      padding-right: unit(@desktop-image-container-width / @base-font-size-px, em);
+      padding-bottom: unit(40px / @base-font-size-px, em); // approximates 45px visually
+      
+      &:after {
+        width: @desktop-image-container-width;
+        background-position: center;
+      }
+    });
+    
+    .lt-ie9 & {
+      // Reduce min-height because IE8 apparently isn't honoring the border-box property,
+      // perhaps due to bug seen here: http://grab.by/JutE
+      min-height: 0;
+      // Fallback for four-value bg-pos syntax
+      &:after {
+        background-position: right center;
+      }
+    }
+  }
+  
+  &_heading {
+    .h2();
+    .respond-to-min(800px, {
+      .h1();
+    });
+  }
```

Comment:
Per @stephanieosan's [comment on the content PR](https://github.com/cfpb/design-manual/pull/362#issuecomment-144751352), please remove the media query here and set `$_heading` to simply `.h1();`